src/main/java/TaskManager.java

```diff
@@ -2,59 +2,65 @@
 
 public class TaskManager {
     public static final String DASH_SEPARATOR = "------------------------------------------------\n";
-    private static int index = 0;
     public static Task[] tasks = new Task[100];
```

Comment:
can create a variable to store 100 so that length can be changed easily by changing the value of the variable